R/plots.R

```diff
@@ -427,9 +463,30 @@ single_emission_rate_plot <- function(sim_dt, puff_dt, output_dt, start_time, en
 #' print(plot)
 #' }
 time_series_plot <- function(sensor_concentrations) {

```

Comment:
Have you defined what `sensor_concentrations` should be (e.g., class, structure/dimensionality, et.c)? Users need to know exactly what should be passed to the function for it to work and get the desired output. 

*Note:* I'd check and include this definition of all required inputs in the roxygen syntax ahead of the function definition. For example, take a look at how I do this in our other plotting functions: https://github.com/Hammerling-Research-Group/puff/blob/73a50573d6ea3d47adb8035a24f8e3a77eeac045/R/plots.R#L7-L14